mu4/palette/internal/palette/palettetree.cpp

```diff
@@ -42,6 +45,13 @@ static QMainWindow* qMainWindow()
     return ioc()->resolve<IMainWindow>("palette")->qMainWindow();
 }
 
+static QColor foregroundColor()
+{
+    using namespace mu::framework;
+    const QmlTheme* theme = UiEngine::instance()->theme();
+    return theme ? theme->fontPrimaryColor() : QColor();
```

Comment:
Hmm, better add something like ITheme, register it in the IoC and use INJECT(... theme)

Reply:
This is a problem. This method is called from a bunch of static methods which are called from static callback passed to libmscore

Reply:
There is INJECT_STATIC, you can consider using it

Reply:
fixed, thanks!

Reply:
Oh, I missed ITheme

Reply:
added ITheme